RPS.html

```diff
@@ -67,27 +67,61 @@
       }
     }
 
+    let playerScore = 0;
+    let computerScore = 0;
+
     function game(playerSelection) {
       const computerSelection = computerPlay();
 
       const resultGameDiv = document.querySelector('.result');
-      const pRoundRPS = document.createElement('p');
-      const pUserSelect = document.createElement('p');
-      const pCompSelect = document.createElement('p');
+      const roundResultParagraph = document.createElement('p');
+      const userChoiceParagraph = document.createElement('p');
+      const computerChoiceParagraph = document.createElement('p');
+      const playerScoreParagraph = document.createElement('p');
+      const computerScoreParagraph = document.createElement('p');
+      
+      const playerRockWin = "You won! Rock beats Scissors.";
+      const playerRockLose = "You lose! Rock beats Scissors.";
+      const playerPaperWin = "You won! Paper beats rock.";
+      const playerPaperLose = "You lose! Paper beats rock.";
+      const playerScissorsWin = "You won! Scissors beats paper.";
+      const playerScissorsLose = "You lose! Scissors beats paper";
+
+      let roundResult = playRound(playerSelection, computerSelection);
+
+      if (roundResult === playerRockWin || roundResult === playerPaperWin || roundResult === playerScissorsWin) {
+        playerScore += 1;
+      } else if (roundResult === playerRockLose || roundResult === playerPaperLose || roundResult === playerScissorsLose) {
+        computerScore += 1;
+      }
 
-      pRoundRPS.textContent = playRound(playerSelection, computerSelection);
-      pUserSelect.textContent = "User selects: " + playerSelection;
-      pCompSelect.textContent = "Comp selects: " + computerSelection;
+      roundResultParagraph.textContent = roundResult;
+      userChoiceParagraph.textContent = "User selects: " + playerSelection;
+      computerChoiceParagraph.textContent = "Comp selects: " + computerSelection;
+      playerScoreParagraph.textContent = "Player score is " + playerScore;
+      computerScoreParagraph.textContent = "Computer score is " + computerScore;
 
-      let textResultGame = [pRoundRPS, pUserSelect, pCompSelect];
+      let textResultGame = [roundResultParagraph, userChoiceParagraph, computerChoiceParagraph, playerScoreParagraph, computerScoreParagraph];
 
       textResultGame.forEach((element) => resultGameDiv.appendChild(element));
+
+      comparisonScores();
+    }
+
+    function comparisonScores() {
+      if (playerScore === 5 && computerScore < 5) {
+        alert("Greetings! You are a winner!");
+        return; 
+      } else if (computerScore === 5 && playerScore < 5) {
+        alert("Shame, you are a loser...");
+        return;
+      }
     }
```

Comment:
First of all, what do you need the returns for?
Second, the conditions on if else if are weird. You don't need what comes after && in the first if. And you don't need the second if at all, just else.

Reply:
I think, return stops function execution

Reply:
well, it does, but there is no code left in the function after that `if`, so it's redundant here.